src/content/api/stats.mdx

```diff
@@ -5,6 +5,7 @@ contributors:
   - skipjack
   - franjohn21
   - byzyk
+  - wizardofhogwarts
```

Comment:
This isn't necessary yet :)

Reply:
ohh yeah!! @EugeneHlushko thanks for figuring out that.